.vscode/settings.json

```diff
@@ -129,5 +129,7 @@
     "whrt",
     "wintel"
   ],
-  "rust-analyzer.cargo.features": []
-}
+  "rust-analyzer.cargo.features": [
+    "s3-integration"
```

Comment:
this should not be in the default features

Reply:
my bad